pkg/handler/k8s.go

```diff
@@ -0,0 +1,51 @@
+package handler
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/netobserv/network-observability-console-plugin/pkg/kubernetes/auth"
+	"github.com/netobserv/network-observability-console-plugin/pkg/kubernetes/resources"
+	"github.com/netobserv/network-observability-console-plugin/pkg/utils"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+func (h *Handlers) GetUDNIdss(ctx context.Context) func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		token, err := auth.GetUserToken(r.Header)
+		if err != nil {
+			writeError(w, http.StatusInternalServerError, err.Error())
+		}
+
+		cudns, err := resources.List(ctx, token, schema.GroupVersionResource{
+			Group:    "k8s.ovn.org",
+			Version:  "v1",
+			Resource: "clusteruserdefinednetworks",
+		})
+		if err != nil {
+			writeError(w, http.StatusInternalServerError, err.Error())
```

Comment:
It would make sense I think to propagate the k8s error code, such as 401/403 ?
I never tried that but I guess it should work:
```
	kerr "k8s.io/apimachinery/pkg/api/errors"
//...
		if err != nil {
			var k8sErr *kerr.StatusError
			if errors.As(err, &k8sErr) {
				writeError(w, int(k8sErr.ErrStatus.Code), err.Error())
			} else {
				writeError(w, http.StatusInternalServerError, err.Error())
			}
		}
```
?

Reply:
done in https://github.com/netobserv/network-observability-console-plugin/pull/688/commits/96a8ce43fb261406127b074f8097a3a50a566bb4